source/player_guide/missions_info/missions/bank_mission.rst

```diff
@@ -9,11 +9,7 @@ Category
 
 .. topic:: Description
 
-   ⚠️ - ⚠️ - ⚠️ - ⚠️ - ⚠️
-
-   🚨 PLACEHOLDER 🚨
-
-   ⚠️ - ⚠️ - ⚠️ - ⚠️ - ⚠️
+This mission is quite rare and map dependent. The mission is looking to see if you are in range of a building called “Land_Offices_01_V1_F” and most cities do not have this building, even fewer maps have it. 
```

Comment:
the description that is under the `topic:: Description` has to be indented once to show it belongs to that topic